scripts/docker-compose.yml

```diff
@@ -1,5 +1,13 @@
 version: '2'
 services:
+  proxy:
+    image: docker.montagu.dide.ic.ac.uk:5000/montagu-reverse-proxy-minimal:mrc-233
```

Comment:
This is a new image which just proxies the API and provides a login page, to mimic a full Montagu deploy. https://github.com/vimc/montagu-proxy/pull/28